.env.shared

```diff
@@ -0,0 +1,4 @@
+# File paths to named pipes
+FOLDER_PATH=/opt/FiberPTS/pipes
```

Comment:
Last change: change to `PIPE_PATH` or `PIPE_FOLDER_PATH` for clarity.

Otherwise, amazing!